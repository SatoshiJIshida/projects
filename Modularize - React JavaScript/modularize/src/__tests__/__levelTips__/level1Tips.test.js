import React from "react";
import { shallow } from "enzyme";
import Level1Tips from "../../components/levelTips/level1Tips";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Level1Tips modals", () => {
  it("should be defined", () => {
    expect(Level1Tips).toBeDefined();
  });

  it("should render correctly", () => {
    const modal = shallow(<Level1Tips name="modal-test" />);
    expect(modal).toMatchSnapshot();
  });
});
