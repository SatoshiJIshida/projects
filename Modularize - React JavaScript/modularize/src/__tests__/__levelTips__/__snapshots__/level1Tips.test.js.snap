// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level1Tips modals should render correctly 1`] = `ShallowWrapper {}`;
